### PR TITLE
Create users with MicroStrategy Web SDK

### PR DESCRIPTION
**Pull request by Jestemrique**
Two methods to create users:
1.- Standard user.
2.- NT authenticated user.

### CODE REVIEW

File: stand-alone/com/microstrategy/samples/subscriptions/CreateScheduleEvent.java [hunk 1]
```diff
@@ -1,49 +0,0 @@
-package com.microstrategy.samples.subscriptions;
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/subscriptions/CreateScheduleEvent.java]: Not sure you meant to delete the "CreateScheduleEvent" sample as part of your pull request?

File: stand-alone/com/microstrategy/samples/sessions/CreateUser.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
+package com.microstrategy.samples.sessions;
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/sessions/CreateUser.java]: I think this sample is more suited for the `com/microstrategy/samples/users` package, as that category is all about reading/editing/creating users. This sessions one is more for authenticating into the platform.

Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/sessions/CreateUser.java]: Done!

File: stand-alone/com/microstrategy/samples/subscriptions/CreateScheduleEvent.java (hunk 1 above)
Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/subscriptions/CreateScheduleEvent.java]: Done.
Can you check, pls?

**Pull request merged**